atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TracingCassandraClient.java

```diff
@@ -90,6 +98,10 @@ public CassandraClient delegate() {
             sink.accept("consistency", consistency_level.name());
             sink.accept("kvs", kvsMethodName);
         })) {
+            log.info(
+                    "multiget_multislice trace ",
+                    SafeArg.of("class", trace.getClass()),
+                    SafeArg.of("trace", trace.toString()));
             return client.multiget_multislice(kvsMethodName, tableRef, keyPredicates, consistency_level);
```

Comment:
Might also be useful to log after this, which will also provide clues as to whether the traceId somehow changed.